tasks/make.js

```diff
@@ -48,7 +48,10 @@ module.exports = function(grunt) {
   // source directory to catch make files included from the primary one.
```

Comment:
Yes, it will no longer descend into directories by default. I'm going to take a smarter approach that will be a better match for something we might do with Composer.